Bottom: fe5037bd7db2e608ea5101812bc3ce5f4ed5ef33
Top:    f5dfb8a41ac68b6a41507f7917a368131fed9ef0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:41:57 +0000

Add reversed orientation field to face3d structure


---

diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
index 4838c43..95232a6 100644
--- a/src/hid/common/face3d.h
+++ b/src/hid/common/face3d.h
@@ -1,5 +1,6 @@
 typedef struct {
   double nx, ny, nz; /* Face normal?*/
+  bool surface_orientation_reversed;
   GList *contours;
 
   /* For cylindrical surfaces */
